app/controllers/api/governments_controller.rb

```diff
@@ -1,28 +0,0 @@
-class Api::GovernmentsController < PublicFacingController
-  skip_before_action :set_cache_control_headers
-  skip_before_action :restrict_request_formats
-  before_action :set_api_cache_control_headers
-  before_action :set_api_access_control_allow_origin_headers
```

Comment:
I think it should now be possible to remove `set_api_cache_control_headers` and `set_api_access_control_allow_origin_headers` from `PublicFacingController`.